src/components/bestCities/styles/BestCitiesStyles.js

```diff
@@ -1,26 +1,42 @@
-import { makeStyles } from '@material-ui/core/styles';
 import styled from 'styled-components';
 
-export const useStyles = makeStyles((theme) => ({
-  root: {
-    marginTop: 52,
-    marginBottom: 52,
-  },
-  titleBar: {
-    background: 'none',
-    textAlign: 'center',
-    fontWeight: 'bold',
-    fontSize: 24,
-  },
-  hover: {
-    '&:hover': {
-      background: '#000',
-      opacity: '0.8',
-    },
-  },
-}));
-
 export const Heading = styled.h2`
+  font-size: 3.8rem;
+`;
+
+export const CityImageContainer = styled.div`
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+`;
+
+export const CityCard = styled.div`
+  margin: 2rem;
+  width: 30rem;
+  font-size: 1.6rem;
+  border-radius: 5px;
   text-align: center;
-  font-size: 38px;
+  box-shadow: 0 1.5rem 4rem rgba(0, 0, 0, 0.3);
+  transition: all 0.3s ease;
+
+  &:hover {
+    transform: translateY(-1.5rem) scale(1);
+    cursor: pointer;
+  }
+
+  img {
+    border-top-left-radius: 5px;
+    border-top-right-radius: 5px;
+    width: 30rem;
+    height: 20rem;
+  }
+
+  p {
```

Comment:
replace this `p` css with  ```
h3 {
    font-size: 1.6rem;
    margin: 0 auto; 
    padding: 1rem;
  }
``` 
You do not have a `p` in` BestCityCard`-> `CityCard` tag.
That will give some margin to the city name under the picture in the card.